man1/aur-fetch.1

```diff
@@ -0,0 +1,38 @@
+.TH AUR-FETCH 1 2018-02-14 AURUTILS
+.SH NAME
+aur-fetch \- fetches packages from a location
+
+.SH SYNOPSIS
+.SY aur
+.B fetch
+.OP -L log_dir
+.OP -t
+.YS
+
+.SH DESCRIPTION
+\fBaur-fetch\fR accepts package names from stdin, and attempts to
+download them.
+
+.SH OPTIONS
+.B \-L
+.RS
+The location of where to store the diff loggs of previously unpacked
```

Comment:
as above, "diff logs" or "diffs". Perhaps we should merge the man pages for `aur-fetch` and `aur-fetch-snapshot`, due to their similarity and latter being auxiliary to former. References to `aur-fetch-snapshot` could be kept in the merged man page.